README.cn.md

```diff
@@ -0,0 +1,118 @@
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+
+<br />
+<p align="center">
+    <a href="https://devchallenge.now.sh/">
+    <img src="https://trello-attachments.s3.amazonaws.com/590fa896d2d25e50583de620/500x500/0bdcc819ea145cb0167619c6d00f2174/D.png" alt="Logo" width="100" height="100">
+  </a>
+  
+  <h3 align="center">DevChallenge</h3>
+
+  <p align="center">
+    开发人员可以通过该网站来提高其通过前端，后端和移动挑战的技能！<br><br>
+    开发人员可以通过前端和后端挑战来提高技能的网站
+       <br />
+    <br />
+     <a href="https://www.devchallenge.com.br/">DevChallenge</a>    
+  </p>
+
+## 指数
+
+- [Índice](#índice)
```

Comment:
Once you change the heading name, you need to change the link too

_Note: To test if it works see [this extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) to preview_